### ISSUE TITLE
Mount XenServer tools and CD image

### ISSUE THREAD

**Issue by dsever** [vatesfr/xo-server#91]
Hi,

In 4.5 user operator can't use XenServer Tools (don't see it at all), also can't mount CD drive from NFS ISO repository. In earlier version it works fine. 

xxx@xxx	XenServer Tools	Operator	
xxx@xxx	NFS_ISO	Viewer

Thanks

Logs from xxx@xxx	NFS_ISO	Viewer:

```
Sep 21 19:24:50 xoa xo-server[30961]: Unauthorized: not authenticated or not enough permissions
Sep 21 19:24:50 xoa xo-server[30961]: at /source/api.js:200:19
Sep 21 19:24:50 xoa xo-server[30961]: at run (/usr/local/lib/node_modules/xo-server/node_modules/babel-runtime/node_modules/core-js/library/modules/es6.promise.js:89:39)
Sep 21 19:24:50 xoa xo-server[30961]: at /usr/local/lib/node_modules/xo-server/node_modules/babel-runtime/node_modules/core-js/library/modules/es6.promise.js:100:28
Sep 21 19:24:50 xoa xo-server[30961]: at flush (/usr/local/lib/node_modules/xo-server/node_modules/babel-runtime/node_modules/core-js/library/modules/$.microtask.js:9:13)
Sep 21 19:24:50 xoa xo-server[30961]: at notify (/usr/local/lib/node_modules/xo-server/node_modules/babel-runtime/node_modules/core-js/library/modules/$.microtask.js:17:13)
Sep 21 19:24:50 xoa xo-server[30961]: at asap (/usr/local/lib/node_modules/xo-server/node_modules/babel-runtime/node_modules/core-js/library/modules/$.microtask.js:45:5)
Sep 21 19:24:50 xoa xo-server[30961]: at notify (/usr/local/lib/node_modules/xo-server/node_modules/babel-runtime/node_modules/core-js/library/modules/es6.promise.js:79:3)
Sep 21 19:24:50 xoa xo-server[30961]: at Object.$reject (/usr/local/lib/node_modules/xo-server/node_modules/babel-runtime/node_modules/core-js/library/modules/es6.promise.js:136:3)
Sep 21 19:24:50 xoa xo-server[30961]: at /usr/local/lib/node_modules/xo-server/node_modules/babel-runtime/node_modules/core-js/library/modules/$.ctx.js:8:17
Sep 21 19:24:50 xoa xo-server[30961]: at tryCatcher (/usr/local/lib/node_modules/xo-server/node_modules/bluebird/js/main/util.js:26:23)
Sep 21 19:24:50 xoa xo-server[30961]: at Promise._settlePromiseFromHandler (/usr/local/lib/node_modules/xo-server/node_modules/bluebird/js/main/promise.js:503:31)
Sep 21 19:24:50 xoa xo-server[30961]: at Promise._settlePromiseAt (/usr/local/lib/node_modules/xo-server/node_modules/bluebird/js/main/promise.js:577:18)
Sep 21 19:24:50 xoa xo-server[30961]: at Promise._settlePromises (/usr/local/lib/node_modules/xo-server/node_modules/bluebird/js/main/promise.js:693:14)
Sep 21 19:24:50 xoa xo-server[30961]: at Async._drainQueue (/usr/local/lib/node_modules/xo-server/node_modules/bluebird/js/main/async.js:123:16)
Sep 21 19:24:50 xoa xo-server[30961]: at Async._drainQueues (/usr/local/lib/node_modules/xo-server/node_modules/bluebird/js/main/async.js:133:10)
Sep 21 19:24:50 xoa xo-server[30961]: at Immediate.Async.drainQueues (/usr/local/lib/node_modules/xo-server/node_modules/bluebird/js/main/async.js:15:14)
```

**Comment by dsever** [comment]
Just update: xotools are not visible using view and operator role, only visible as administrator, but can't be mounted. CD media visible in any cases, but can't be mounted.

**Comment by olivierlambert** [comment]
This is fixed and we'll be release in the 4.6 very soon :)
